content/user/payment/square.md

```diff
@@ -7,16 +7,44 @@ weight: 10
 
 There are two Square modules for Zen Cart: 
 - A newer one, based on the Square Web Payments API 
-- An older one, based on SqPaymentForm, which will be deprecated in 2022
+- An older one, based on SqPaymentForm, which will be deprecated in July 2022.
 
 Square users will need to move to the new [Square Web Payments API plugin](https://www.zen-cart.com/downloads.php?do=file&id=2345). 
 
 ## Newer Integration - Square API 
 The newer Square integration is available in the Plugins library as [Square Web Payments API plugin](https://www.zen-cart.com/downloads.php?do=file&id=2345). 
 
-Please note that the newer integration requires PHP 7.4, so you will need
-to be running Zen Cart 1.5.7 or higher. If you need to upgrade, please 
-review the [upgrading instructions](/user/upgrading/). 
+Please note that, while Square suggests PHP 7.4 or greater, we have found this mod will work with PHP 7.3 and Zen Cart version 1.5.6c. That said, upgrading to the current version of Zen Cart and at least PHP 7.4 is recommended for security. If you need to upgrade, please review the [upgrading instructions](/user/upgrading/). 
+
+### Setup Instructions: 
```

Comment:
these instructions seem to be a migration from the older plugin as opposed to setting up a new version.

in addition, i am not sure why we are not updating this page:
https://www.zen-cart.com/content.php?349-Square-Payments-for-Zen-Cart

Reply:
We're trying to deprecate the old content.php pages.  I'm going to link out to the documentation site from this page.  Instructions will live on the docs site moving forward.